.github/workflows/windows.yml

```diff
@@ -27,5 +27,6 @@ jobs:
         run: |
           $Env:GOPATH = "$Env:GITHUB_WORKSPACE"
           cd "$Env:GITHUB_WORKSPACE"
-          cd "src/github.com/aws/amazon-ecs-agent"
-          go test -race -tags unit -timeout 40s ./agent/...
\ No newline at end of file
+          cd "src/github.com/aws/amazon-ecs-agent/agent"
+          $packages=go list .\... | Where-Object {$_ -NotMatch 'vendor'}
+          go test -v -tags unit -timeout=40s $packages
```

Comment:
why was `-race` removed here?

Reply:
I was following https://github.com/aws/amazon-ecs-agent/blob/master/scripts/run-unit-tests.ps1 but I can add it back.